src/apps/frontend/components/header/user-menu-dropdown.component.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+type UserMenuDropdownProps = {
+  dropdownOpen: boolean;
+  dropdownRef: React.RefObject<HTMLDivElement>;
+  logout: () => void;
+  setDropdownOpen: (dropdownOpen: boolean) => void;
+};
```

Comment:
I am not sure how Tailwind admin is setup, but we want a setup like `SidebarMenuItem` where more items can be without chaging the dropdown markup or hardcoding about one particular type of dropdown menu item